deploy/vagrant/conf/ec2.yml.template

```diff
@@ -11,7 +11,7 @@ Key_Path: "~/.ssh/aws-east.pem"
 # When using spark, we recommend "r3.large", compile spark needs 2G memory at least
 #
 # Some Instance Type must need PV/HVM(refer to AMI below)
-Instance_Type: "t2.small"
+Instance_Type: "r3.large"
```

Comment:
only `t2.micro` is free, but it only has 1G mem, even running basic Tachyon(10MB worker mem) + HDFS2 on it causes OOM .